script.js

```diff
@@ -0,0 +1,22 @@
+function calculate() {
+      var num1 = parseInt(document.getElementById('num1').value)
+      var num2 = parseInt(document.getElementById('num2').value)
+      var operator = document.getElementById('operator').value
+
+      document.getElementById('answer').innerHTML=calculateAnswer(operator, num1, num2)
+}
+    
+function calculateAnswer(operator, num1, num2) {
+  switch (operator) {
+    case '+':
+      return num1 + num2;
+    case '-':
+      return num1 - num2;
+    case '*':
+      return num1 * num2;
+    case '/':
+      return num1 / num2;  
```

Comment:
You should have no semicolons ending these lines. Pick one way to end lines and stick with it. Semicolons will be incorrect as soon as we start using eslint, but even before that you need to be consistent.